src/main/java/com/sessions/session7/TemaSesiune7.java

```diff
@@ -0,0 +1,73 @@
+package com.sessions.session7;
+
+import java.util.ArrayList;
+
+public class TemaSesiune7
+{
+
+    public static void main(String[] args)
+    {
+        //1.Create a shopping list with Array and print the values
+
+        System.out.println("Exercitiu 1:");
+
+        String[] ShoppingList={"Apples","Banans","Oil","Sugar","Bred","Water"};
+        for(int i=0;i<ShoppingList.length;i++)
+        {
+            System.out.println(ShoppingList[i]);
+        }
+        System.out.println(" ");
+
+
+        //2.Create a wishlist for christmas with ArrayList and print the values
+
+        System.out.println("Exercitiu 2:");
+
+        ArrayList WishList=new ArrayList();
```

Comment:
If you know the objects inside will be String ojects, you should specify that when declaring the list. for example:
  List<String> studentList = new ArrayList<>();
  in this way you can only add Strings and not other type of objects.